### PR TITLE
Add scripts to export data from ES testing cluster and to import the data locally

### PR DESCRIPTION
**Pull request by rockdaboot**
A helper to test locally with exported data from the ES testing cluster.

### CODE REVIEW

File: src/plugins/profiling/scripts/export_from_testing.sh [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+#  export_events "profiling-events-5pow$i" "$date_from" "$date_to"
+#done
+
+# We need all stacktraces, stackframes and executables as 'LastSeen' may be outside [date_from, date_to].
+#export_index "profiling-stacktraces"
+#export_index "profiling-stackframes"
+export_index "profiling-executables"
+
+echo -e '
+ Import the data with ...
```
Comment (rockdaboot) [inline src/plugins/profiling/scripts/export_from_testing.sh:114]: This line needs to be extended once the import script is ready.

**Review by inge4pres — COMMENTED** [review]
Thanks for the automate-all-the-things approach 😄 
Some comments below

File: src/plugins/profiling/scripts/export_from_testing.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (rockdaboot) [reply src/plugins/profiling/scripts/export_from_testing.sh:114]: See my comment above

File: src/plugins/profiling/scripts/export_from_testing.sh [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    *)
+      usage
+      ;;
+    esac
+done
+
+[[ "$date_to" -le "$date_from" ]] && die "Invalid time range"
+
+# Pull latest docker image
+docker pull elasticdump/elasticsearch-dump
```
[the remark this answers is not in the document]
Comment (inge4pres) [reply src/plugins/profiling/scripts/export_from_testing.sh:98]: The risk that a `latest` is not compatible with the ES version we use (possibly the latest) is minimal, but in my experience these failures are the most painful to detect.
If you want to stick with `latest` there's no need to add the tag, it's implicit.

[the remark this answers is not in the document]
Comment (rockdaboot) [reply src/plugins/profiling/scripts/export_from_testing.sh:98]: IMO, the tool itself is not bound to a specific ES version (or is it ?). In fact, it should just work with any supported ES version. If not: the tool is our code and needs to be fixed when that ever happens.

Adding `:latest` makes it explicitly clear that we want to use the latest version and nothing else.

File: src/plugins/profiling/scripts/export_from_testing.sh (hunk 1 above)
Comment (rockdaboot) [reply src/plugins/profiling/scripts/export_from_testing.sh:114]: Done

File: src/plugins/profiling/scripts/export_from_testing.sh [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  fi
+}
+
+export_events() {
+  local index=$1
+  local from=$2
+  local to=$3
+
+  docker run --rm -ti --net=host -v "$PWD:/data" -w /data elasticdump/elasticsearch-dump \
+    --input="https://""${ELASTIC_TESTING_CREDENTIALS}""@profiling-es.35.240.6.93.ip.es.io/${index}" \
```
[the remark this answers is not in the document]
Comment (inge4pres) [reply src/plugins/profiling/scripts/export_from_testing.sh:39]: yeah not a big deal keeping it unique, I am just not a fan of colon in env vars 😄

File: src/plugins/profiling/scripts/import_from_testing.sh [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    dir="."
+  fi
+
+  pushd "$dir"
+  docker run --rm -ti --net=host -v "$PWD:/data" elasticdump/elasticsearch-dump:latest \
+    --input="/data/$file" \
+    --output="http://admin:changeme@localhost:9200/$index" \
+    --type=data --fsCompress --noRefresh --support-big-int --limit=10000
+  popd
+}
```
Comment (inge4pres) [inline src/plugins/profiling/scripts/import_from_testing.sh:37]: Did we agree on adding the mappings too when exporting and re-importing?
Will you be adding that in this PR?

Comment (rockdaboot) [reply src/plugins/profiling/scripts/import_from_testing.sh:37]: I'd say we should create another PR. Adding mappings has some edges that needs to be thought of. And except for my stupid mistake today, we possibly don't need them right now.

**Review by inge4pres — APPROVED** [review]

**Pull request merged**